snuba/replacers/errors_replacer.py

```diff
@@ -203,9 +207,17 @@ def process_message(
             raise InvalidMessageType("Invalid message type: {}".format(type_))
 
         if processed is not None:
-            bypass_projects = get_config("replacements_bypass_projects", "[]")
-            projects = json.loads(cast(str, bypass_projects))
-            if processed.get_project_id() in projects:
+            manual_bypass_projects = get_config("replacements_bypass_projects", "[]")
+            auto_bypass_projects = list(
+                get_config_auto_replacements_bypass_projects(datetime.now()).keys()
+            )
+            projects_to_skip = auto_bypass_projects
+            if manual_bypass_projects is not None:
+                try:
+                    projects_to_skip.extend(ast.literal_eval(manual_bypass_projects))
```

Comment:
You basically never want to `ast.literal_eval` code anywhere. `json.loads` will do just fine here